internal/controller/filesystem_helpers.go

```diff
@@ -227,9 +232,14 @@ func newLvmBlockDevice(ctx context.Context, c client.Client, nnfNodeStorage *nnf
 		return nil, dwsv1alpha2.NewResourceError("could not get logical volume name").WithError(err).WithMajor()
 	}
 
+	percentVG := 100
```

Comment:
Shouldn't this be pulled from the profile?

Reply:
This percentVG value is used to fill in the $PERCENT_VG variable in the "lvcreate" command for the "--extents" option. If there's only one LV on a VG, "$PERCENT_VG" will expand to "100%VG" so that the LV gets all the space on the VG. If they wanted something other than that (I'm not sure that they ever would), they could use a hard coded value other than "$PERCENT_VG". If the VG is shared between multiple LVs, then "$PERCENT_VG" will expand to something like "25%VG" if there were 4 LVs on the VG. The actual percent of the VG depends on the number of compute nodes making an allocation on a Rabbit, and it may differ between Rabbits in a single workflow.